var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -226,7 +226,8 @@ class Paraview(CMakePackage, CudaPackage, ROCmPackage):
     depends_on("protobuf@3.4:3.18", when="@:5.10%xl_r")
     # protobuf requires newer abseil-cpp, which in turn requires C++14,
     # but paraview uses C++11 by default
-    depends_on("protobuf@3.4:3.21", when="@:5.11")
+    depends_on("protobuf@3.4:3.21", when="@:5.12")
```

Comment:
I don't understand this change. I think if this is going to be true it seems always, then maybe we can just drop the version constraint?

Reply:
I made it for 5.11+ and added a note to check it when ParaView is no longer C++11.